core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -466,15 +470,12 @@ private void renderSidebar() {
       }
 
       for (Competitor competitor : sortedCompetitors) {
-        if (!firstTeam) {
+        if (!(firstTeam || (isWool() && competitorsWithGoals.size() > 5))) {
           // Add a blank row between teams
           rows.add("");
         }
         firstTeam = false;
 
-        // Add a row for the team name
-        rows.add(competitor.getStyledName(NameStyle.FANCY).render().toLegacyText());
```

Comment:
I really do not think that this should be changed.
That not all teams & their wools will fit is a shame, but joining team name & objectives is a worse solution than the problem itself, i'd just accept that the last teams' wools (wich, because they should be sorted, should be the losing one) won't be visible in scoreboard and only visible in /match